### DISCUSSION TITLE
Can bad ffmpeg break Wyze?

### DISCUSSION THREAD

**Discussion by Tsardoz** [gtxaspec/wz_mini_hacks#677] [Q&A] [answered]
I had go2rtc working fine at 192.168.0.140:1984 but decided the image quality could be a bit better so started frigging around with the yaml ffmpeg settings. Nothing I was doing worked so went back to the original setup. To my surprise I could no longer see the streaming server! Nothing ... I can log in via ssh and the phone app works too. I even reset the firmware, loaded up a new SD card to reset everything to default. Configured the go2rtc in the config file. Nothing. All I had to do was this first time and it worked. GO2RTC_SERVER_ENABLED="true"
GO2RTC_HI_RES_ENABLED="true"
GO2RTC_HI_RES_ENABLE_AUDIO="true"
GO2RTC_LOW_RES_ENABLED="true"
GO2RTC_LOW_RES_ENABLE_AUDIO="true"

Now it doesn't. Conclusion is I bricked it but I do not know how this could have happened. Any thoughts? Using 4.36.9.139

**Comment by Tsardoz** [comment, marked as answer]
OK I am not sure what I did wrong but the answer is no. I got a new SD card and new camera (I have 2). Set it all up from scratch and got go2rtc working. Then transferred the SD card to original camera and it works fine. So either there was a card fault or I did something wrong in the setup of it. No need to panic.

**Reply by Answer-1** [reply]
I got something similar a couple times, go2rtc stopped working. Even had instances where the cam was booting the Wyze stock firmware. Got the SD card out, did a quick format, put the files back on the SD card, and once I put the card back in the cam, it works like it was before. Seems like something is happening to the SD cards somehow (Windows reports and error and proposes to fix it) which prevents the cam from working correctly.

**Reply by clee604** [reply]
I've seen this happen too as well. I've noticed sometimes in the record folder there's a folder with a date WAY in the past. Not sure if that bricks it or if it's something bad in the tmp folder. Flushing everything out of the sd card and then recopying wz_mini seems to fix it.

Not sure if it is because the camera was powered off while it was still recording or something else, but I've noticed it a few times on a few different cameras now.

**Reply by Answer-1** [reply]
My cams are pretty much all set to not record on the SD so in my case, I don't think it's writing a recording while powered off.
I just tried to disable swap to see how they're gonna act, maybe this has something to do with the SD going partially corrupt.
